src/utils/installPackage.js

```diff
@@ -0,0 +1,51 @@
+const {spawn} = require('child_process');
+const config = require('./config');
+
+module.exports = async function(dir, name) {
+  let yarn;
+  try {
+    yarn = await config.resolve(dir, ['yarn.lock']);
```

Comment:
maybe instead of trying to detect yarn based on yarn.lock we should just look for the command?

Reply:
This sounds like a more solid approach, will change it